frontend/public/components/app.jsx

```diff
@@ -374,25 +375,32 @@ const PollConsoleUpdates = React.memo(function PollConsoleUpdates() {
   const prevPluginManifestsData = prevPluginManifestsDataRef.current;
   const stateInitialized = _.isEmpty(updateError) && !_.isEmpty(prevUpdateData);
 
-  const pluginsListChanged = !_.isEmpty(_.xor(prevUpdateData?.plugins, updateData?.plugins));
+  const newPluginsList = _.xor(prevUpdateData?.plugins, updateData?.plugins);
```

Comment:
I think this will attempt to fetch the manifest for *removed* plugins as well, which we don't want.